src/main/java/org/opentestsystem/saaif/item/SaaifItemConstants.java

```diff
@@ -43,20 +43,23 @@
      * The SAAIF item formats.
      */
     interface ItemFormat {
-        String FORMAT_SHORT_ANSWER = "sa";
+        String FORMAT_SA = "sa";
```

Comment:
I like it.  Would there be a factory or something that would receive a type like 'sa' or 'wer' and  decide which concrete implementation to return like SaItemProperties and WerItemProperties, and I'm assuming there is an interface these concrete class implement like ItemProperties.   

ItemProperties would have methods like getResponseTypes, getPageLayout.  

I wonder if these concrete classes could return things like a default, minimum attribute list, effectively replacing what I'm doing now in SaaifItemFactory.  SaaifItemFactory is going to get huge given how many types we have so it makes sense to reconsider what I'm doing here.